pootle/apps/pootle_app/management/commands/create_project.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+import os
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import BaseCommand
+
+from pootle_language.models import Language
+from pootle_project.models import Project
+
+
+class Command(BaseCommand):
+    help = "Creates a new project from command line."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--code',
+            action='store',
+            dest="code",
+            help='Project code',
+        )
+        parser.add_argument(
+            "--name",
+            action="store",
+            dest="name",
+            help="Project name",
+        )
+        parser.add_argument(
+            "--quality",
+            action="store",
+            dest="quality",
+            default="standard",
+            help="Quality management",
+        )
+        parser.add_argument(
+            "--filetype",
+            action="store",
+            dest="filetype",
+            help="File type: [ po/pot | xliff | xlf | ts | lang ]",
+        )
+        parser.add_argument(
+            "--structure",
+            action="store",
+            dest="structure",
+            default="gnu",
+            help="Catalogue structure: [ automatic | gnu | no-gnu | filesystem ]",
```

Comment:
We have used `non-gnu` for years, so I would stick to it.

Do we want `automatic`?

What is `filesystem`?

Reply:
these need to match the preset mappings, or be an actual mapping

Reply:
If I am correct these are the supported preset mappings https://github.com/translate/pootle/blob/538300f9e13cc18f54f79644819995a9e5511334/pootle/apps/pootle_fs/forms.py#L23-L31 So `gnu`, `non-gnu`, `django`.

Reply:
yep

Reply:
but best not to hardcode - ie use `FS_CHOICES.keys()` or similar

Reply:
I agree, provided the `custom` key is dropped.

Reply:
hmm, we need to figure out how to handle translation_mappings, which would be the `custom` option in  terms of presets